fog/sample-paykit/src/cached_tx_data/mod.rs

```diff
@@ -409,8 +409,10 @@ impl CachedTxData {
         }
 
         self.missed_block_ranges.extend(new_missed_block_ranges);
-        let fog_common_block_ranges: Vec<fog_common::BlockRange> =
-            CachedTxData::convert_block_ranges(&self.missed_block_ranges);
+        let fog_common_block_ranges: Vec<fog_common::BlockRange> = &self
+            .missed_block_ranges
+            .iter()
+            .collect::<Vec<fog_common::BlockRange>>();
```

Comment:
You probably don't need type annotations twice, `Vec<fog...>` can appear once in either place and be inferred in the other